lib/tasks/post_precompile_assets.rake

```diff
@@ -0,0 +1,8 @@
+task after_assets_precompile: :environment do
+  # Runs postcss/autoprefixer to ensure that our CSS works for all the browsers we support.
+  system("yarn postcss")
+end
+
+Rake::Task["assets:precompile"].enhance do
+  Rake::Task["after_assets_precompile"].execute
```

Comment:
This should do it @mstruve. I ran it locally and checked that it runs after now. Good catch and thanks!